doc/user/content/sql/explain-plan.md

```diff
@@ -241,6 +241,10 @@ Below the plan, a "Used indexes" section indicates which indexes will be used by
 ### Reference: Plan operators
 
 {{< tabs >}}
+{{< tab "In fully optimized LIR plans" >}}
+{{< explain-plans/operator-table data="explain_plan_operators" planType="lir" >}}
```

Comment:
Are we talking about EXPLAIN PHYSICAL PLAN for VIEW/MATERIALIZED VIEW here, yes? If so, I think we should go with the `PHYSICAL plans`.